Okta.AspNetCore.Test/OpenIdConnectOptionsHelperShould.cs

```diff
@@ -13,15 +13,17 @@
 using NSubstitute;
 using Okta.AspNet.Abstractions;
 using Xunit;
+using Microsoft.AspNetCore.Authentication.OAuth.Claims;
+using System.Security.Claims;
 
 namespace Okta.AspNetCore.Test
 {
     public class OpenIdConnectOptionsHelperShould
     {
         [Theory]
-        [InlineData(true)]
-        [InlineData(false)]
-        public void SetOpenIdConnectsOptionsCorrectly(bool getClaimsFromUserInfoEndpoint)
+        [InlineData(true, true)]
+        [InlineData(false, false)]
```

Comment:
what about (false, true) and (true, false)?

Reply:
will create another test for that